presto-tests/src/test/java/io/prestosql/execution/TestEventListenerWithSplits.java

```diff
@@ -90,7 +91,7 @@ protected QueryRunner createQueryRunner()
         queryRunner.getCoordinator().getResourceGroupManager().get()
                 .setConfigurationManager("file", ImmutableMap.of("resource-groups.config-file", getResourceFilePath("resource_groups_config_simple.json")));
 
-        queries = new EventsAwaitingQueries(generatedEvents, queryRunner);
+        queries = new EventsAwaitingQueries(generatedEvents, queryRunner, Duration.ofSeconds(1));
```

Comment:
if you added 1s everywhere, then maybe we do not need this parametric and we can do 1s unconditionally?

Reply:
Maybe. Let's leave it though. There are not many calls to this constructor.